packages/amplify-provider-awscloudformation/src/aws-predictions-regions.js

```diff
@@ -116,6 +121,7 @@ const Comprehend = {
   'eu-west-3': 'eu-west-1',
   'eu-central-1': 'eu-central-1',
   'eu-north-1': 'eu-central-1',
+  'eu-south-1': 'eu-central-1',
```

Comment:
Comprehend is not available in eu-south-1. Falling back to eu-central-1.

https://docs.aws.amazon.com/general/latest/gr/comprehend.html